test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -626,6 +636,92 @@ public void notSettingStatusDoesntClearStatus() {
         verify(account, never()).setStatus(any());
     }
 
+    @Test
+    public void updateParticipantEmailCannotBeChanged() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval();
+        when(account.getEmail()).thenReturn("persisted@email.com");
+        when(account.getEmailVerified()).thenReturn(Boolean.TRUE);
+        
+        StudyParticipant participant = new StudyParticipant.Builder().copyOf(PARTICIPANT)
+                .withEmail("updated@email.com").withEmailVerified(Boolean.FALSE).build();
+        
+        participantService.updateParticipant(STUDY, ImmutableSet.of(), participant);
+        
+        verify(accountDao).updateAccount(accountCaptor.capture());
+        Account account = accountCaptor.getValue();
+        verify(account, never()).setEmail("updated@email.com");
+        verify(account, never()).setEmailVerified(Boolean.FALSE);
+        verify(accountWorkflowService, never()).sendEmailVerificationToken(STUDY, ID, "updated@email.com");
```

Comment:
Is it worthwhile to add tests for

* deleting email address?
* email address unchanged?

Reply:
Added tests of deleting email or phone (you can't). We have updateParticipantEmailCannotBeChanged() to test changing of email (and same with phone).